### PR TITLE
105 Docs  - Homepage style updates

### LINKED ISSUE #105: Docs - Homepage style updates
- fix background image opacity and dark mode color issue
- fix dark mode Card text color issue
- fix Card text overflow issue
- Make hero header text font size bigger

### PR DESCRIPTION
**Pull request by jaygiang**
issue #105 

- Fixes card text color in dark mode
- Added media queries to handle Hero header and subheader
- Fixes hero banner background image color and adds opacity
- Fixes cards overflow on smaller screens

https://github.com/user-attachments/assets/28c00bbe-8072-47fb-8986-a5474d1aa335

### CODE REVIEW

File: src/pages/index.module.css [hunk 1]
```diff
@@ -8,18 +8,76 @@
   text-align: center;
   position: relative;
   overflow: hidden;
+  background-color: #205493;
```
Comment (JCopeland-PIFSC) [inline src/pages/index.module.css]: This color code maps to `--ifm-color-primary:`. We should use the color variable rather then the value.

File: src/components/HomepageFeatures/styles.module.css [hunk 2]
```diff
@@ -43,10 +47,11 @@
   box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
   margin-bottom: 20px;
   transition: transform 0.2s ease-in-out;
-  height: 300px;
   display: flex;
   flex-direction: column;
   justify-content: space-between;
+  color: #000;
```
Comment (JCopeland-PIFSC) [inline src/components/HomepageFeatures/styles.module.css]: When color values map to specific color, the color name should be used.
`color: black;`

File: src/pages/index.module.css [hunk 3]
```diff
@@ -8,18 +8,76 @@
...
+.heroHeading {
+  font-size: 2.5rem;
+}
+
+.heroSubHeading {
+  font-size: 1.2rem;
+}
+
+.heroTextColor {
+  color: #fff;
```
Comment (JCopeland-PIFSC) [inline src/pages/index.module.css]: When color values map to specific color, the color name should be used.
`color: white;`

**Review by JCopeland-PIFSC — CHANGES_REQUESTED** [review]
I have a couple of small changes I think should be made for consistency.

File: src/pages/index.module.css (hunk 1 above)
Comment (jaygiang) [reply src/pages/index.module.css]: Yeah, initially I set that as `--ifm-color-primary:`, but it actually changes to Dark mode's `--ifm-color-primary` which is `#25c2a0`. https://github.com/NMFS-RADFish/radfish/blob/main/src/css/custom.css#L27

Screenshot 2024-09-26 at 4 08 08 PM

File: src/components/HomepageFeatures/styles.module.css (hunk 2 above)
Comment (jaygiang) [reply src/components/HomepageFeatures/styles.module.css]: Thanks! Changes made here: https://github.com/NMFS-RADFish/radfish/pull/106/commits/dc9b9cc887c7dbb2c5be6fa2aa397ac8d20aa7a7

File: src/pages/index.module.css (hunk 3 above)
Comment (jaygiang) [reply src/pages/index.module.css]: Changes made here: https://github.com/NMFS-RADFish/radfish/pull/106/commits/dc9b9cc887c7dbb2c5be6fa2aa397ac8d20aa7a7

File: src/pages/index.module.css (hunk 1 above)
Comment (jaygiang) [reply src/pages/index.module.css]: I added a `radfish-primary-blue` to be used globally. It works in light and dark mode, I think cause it's not defined under `[data-theme="dark"]`.  Lmk if you think this is a good method - [37c0d55](https://github.com/NMFS-RADFish/radfish/pull/106/commits/37c0d5507fda4647c3ec2618088e0dbdbbabfcec)

Comment (JCopeland-PIFSC) [reply src/pages/index.module.css]: 👍 good solution.

**Pull request merged**